docs/readme/readme_pt_br.md

```
@@ -89,21 +89,13 @@ Ou qualquer sistema de controle de versão de sua escolha.
-#### Apoiadores Silver
-
-<img src="../../assets/images/sponsors/commit-company.png" width="70"/>
-
-#### Apoiadores Bronze
-
-<a href="https://zuplo.link/bruno">
-    <img src="../../assets/images/sponsors/zuplo.png" width="120"/>
-</a>
+<div align="center">
+    <a href="https://github.com/usebruno/bruno/graphs/contributors">
+        <img src="https://contrib.rocks/image?repo=usebruno/bruno" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)